src/components/cluster/detail/ClusterApps.js

```diff
@@ -190,15 +199,29 @@ class ClusterApps extends React.Component {
     });
   };
 
+  // userInstallApps returns a list of just the apps that the user installed
+  // since the list of apps in a cluster also includes apps that were installed
+  // automatically.
+  userInstalledApps = () => {
```

Comment:
Since this is a function, can you please add a verb at the beginning of its name (such as `getUserInstalledApps`)?